Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -174,4 +176,18 @@ namespace Hazel {
 		return m_Data.VSync;
 	}
 
+	void WindowsWindow::SetIcon(const std::string& path)
+	{
+		int width, height, channels;
+		stbi_set_flip_vertically_on_load(1);
```

Comment:
I'm quite sure this code is the same as the one used for texture loading. 
A good approach would be to extract a helper function (i.e. _load_image_) with parameters to flip on load and use that both here and in the texture class.

Reply:
Yes, but I will need to create new utils.h file where I put code like this. Right now it will be copy pasted.

Reply:
That's exactly what I would do move it to a helper file/class, a few more clicks for better maintainability.